examples/chart/teleport/scripts/create-internal-pki

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
```

Comment:
same here

Reply:
Okay then I'll move these scripts to my own repo, along with the steps to test Teleport locally on minikube 👍

Reply:
> along with the steps to test Teleport locally on minikube

I'm saying this because `create-internal-pki` is a requisiet for `install-on-minikube`. The latter alone doesn't create a testable Teleport installation because we can't `tsh login` into it due to https://github.com/gravitational/teleport/pull/2014#issuecomment-399619312

Reply:
Or do you want me to leave this script along with `install-on-minikube`?